null_statsd.gemspec

```diff
@@ -1,25 +1,26 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'null_statsd/version'
+require "null_statsd/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = "null_statsd"
-  spec.version       = NullStatsd::VERSION
-  spec.authors       = ["Imad Mouaddine"]
-  spec.email         = ["imouaddine@usertesting.com"]
+  spec.name = "null_statsd"
+  spec.version = NullStatsd::VERSION
+  spec.authors = ["Imad Mouaddine"]
+  spec.email = ["imouaddine@usertesting.com"]
 
-  spec.summary       = %q{Implements null pattern for Statsd client}
-  spec.description   = %q{Implements null pattern for Statsd client}
-  spec.homepage      = "https://github.com/usertesting/null_statsd"
-  spec.license       = "MIT"
+  spec.summary = %q{Implements null pattern for Statsd client}
+  spec.description = %q{Implements null pattern for Statsd client}
+  spec.homepage = "https://github.com/usertesting/null_statsd"
+  spec.license = "MIT"
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.files = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
```

Comment:
Might want to also add the new `UserTesting.png` in here, so that downloading the `.gem` doesn't have the image included